src/VisualStudio/IntegrationTest/TestUtilities/InProcess/TextViewWindow_InProc.cs

```diff
@@ -278,6 +278,17 @@ protected Action<CancellationToken> GetExecuteOnActionViewCallback(Action<IWpfTe
                 action(view);
             };
 
+        public void InvokeQuickInfo()
+        {
+            ThreadHelper.JoinableTaskFactory.Run(async () =>
+            {
+                await ThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
```

Comment:
@sharwell Why the SwitchToMainThreadAsync here if the GetQuickInfo doesn't have that?

Reply:
For the call to `GetActiveTextView()`

Reply:
GetQuickInfo also has it, but it's implicit in the `ExecuteOnActiveView` call.